plugins/org.jkiss.dbeaver.erd.ui/plugin.xml

```diff
@@ -500,12 +506,15 @@
         <menuContribution allPopups="false" locationURI="menu:dataSourceMenu?after=tools">
             <command commandId="org.jkiss.dbeaver.erd.diagram.create" label="%menu.create.new.diagram">
                 <visibleWhen checkEnabled="true">
-                    <with variable="selection">
-                        <count value="+"/>
-                        <iterate operator="and">
-                            <adapt type="org.jkiss.dbeaver.model.struct.DBSEntity"/>
-                        </iterate>
-                    </with>
+                    <and>
+                        <with variable="selection">
+                            <count value="+"/>
+                            <iterate operator="and">
+                                <adapt type="org.jkiss.dbeaver.model.struct.DBSEntity"/>
+                            </iterate>
+                        </with>
+                        <test property="org.jkiss.dbeaver.core.global.hasPermission" value="database-developer"/>
```

Comment:
Move <test> tag into <adapt> and remove <and>. Please do it for all similar cases.